src/cve/pipeline/input.py

```diff
@@ -200,6 +201,20 @@ async def _inner():
     @catch_pipeline_errors
     @traced(module="input")
     @stage
+    def calculate_intel_scores(message: AgentMorpheusEngineInput) -> AgentMorpheusEngineInput:
+        trace_id.set(message.input.scan.id)
+
+        intel_scorer = IntelScorer(run_config, message.input.scan.id)
+
+        message.info.intel = [intel_scorer.calculate_intel_score(intel) for intel in message.info.intel]
+
+        return message
+
+    if run_config.general.generate_intel_score:
+        pipe.add_stage(calculate_intel_scores(config))
+
+    @catch_pipeline_errors
+    @stage
```

Comment:
Need to re-add @traced decorator to this function stage.
```suggestion
    @catch_pipeline_errors
    @traced(module="input")
    @stage
```